src/site/content/en/blog/serial/index.md

```diff
@@ -602,12 +602,12 @@ Aeroplane factory photo by [Birmingham Museums Trust] on [Unsplash].
 [Betaflight Configurator]: https://github.com/betaflight/betaflight-configurator
 [Espruino Web IDE]: http://espruino.com/ide
 [Microsoft MakeCode]: https://www.microsoft.com/en-us/makecode
-[explainer]: https://github.com/reillyeon/serial/blob/gh-pages/EXPLAINER.md
-[spec]: https://reillyeon.github.io/serial/
+[explainer]: https://github.com/reillyeon/serial/blob/main/EXPLAINER.md
+[spec]: https://github.com/reillyeon/serial/
```

Comment:
These two links should probably go to the WICG repo that reillyeon's repo is forked from.